pkg/reconciler/channel/channel.go

```diff
@@ -153,3 +188,65 @@ func (r *Reconciler) updateStatus(ctx context.Context, desired *v1alpha1.Channel
 
 	return c, err
 }
+
+// reconcileBackingChannel reconciles Channel's 'c' underlying CRD channel.
+func (r *Reconciler) reconcileBackingChannel(ctx context.Context, resourceClient dynamic.ResourceInterface, c *v1alpha1.Channel) (*duckv1alpha1.Channelable, error) {
+
+	channel, err := resourceClient.Get(c.Name, metav1.GetOptions{})
+	channelable := &duckv1alpha1.Channelable{}
+
+	// If the resource doesn't exist, we'll create it
+	if err != nil {
+		if apierrs.IsNotFound(err) {
+			newChannel, err := resources.NewChannel(c)
+			if err != nil {
+				logging.FromContext(ctx).Error("Failed to create Channel from ChannelTemplate", zap.Any("channelTemplate", c.Spec.ChannelTemplate), zap.Error(err))
+				return nil, err
+			}
+			logging.FromContext(ctx).Debug(fmt.Sprintf("Creating Channel CRD Object: %+v", newChannel))
```

Comment:
We log every possible outcome, skip this log.

Reply:
Done